src/main/java/com/oltpbenchmark/benchmarks/tpcc/procedures/Delivery.java

```diff
@@ -34,53 +34,66 @@ public class Delivery extends TPCCProcedure {
     private static final Logger LOG = LoggerFactory.getLogger(Delivery.class);
 
     public SQLStmt delivGetOrderIdSQL = new SQLStmt(
-            "SELECT NO_O_ID FROM " + TPCCConstants.TABLENAME_NEWORDER +
-            " WHERE NO_D_ID = ? " +
-            "   AND NO_W_ID = ? " +
-            " ORDER BY NO_O_ID ASC " +
-            " LIMIT 1");
+        """
+            SELECT NO_O_ID FROM %s
+             WHERE NO_D_ID = ?
+               AND NO_W_ID = ?
+             ORDER BY NO_O_ID ASC
+             LIMIT 1
+        """.formatted(TPCCConstants.TABLENAME_NEWORDER));
```

Comment:
:thumbsup: :heart:

Reply:
Strictly speaking these aren't related, right?  So could split this out to a separate PR, but I won't force it ;)

Reply:
Yeah, I was playing around with it separately. Let me split it out, it's the right thing to do.